### PR TITLE
Create contact form section

### PR DESCRIPTION
**Pull request by nicupop729**
## Web-Designer-Portfolio

In this branch we have introduced the following changes:

- Created Contact Form;
- Styled Contact Form;

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,6 +268,74 @@ <h4 class="experience-title">Skills</h4>
...
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea maxlength="500" name="message" required>
+Enter text here</textarea
```
Comment (ZeenatLawal) [inline index.html]: - [x] The Enter text here in the `textarea` input should be a placeholder as specified in the requirements.

File: index.html [hunk 2]
```diff
@@ -268,6 +268,74 @@ <h4 class="experience-title">Skills</h4>
...
+        >
+        <button type="submit">Get in touch</button>
+      </form>
```
Comment (ZeenatLawal) [inline index.html:292]: - [x] The Get in touch button should follow the button styling: Active, Hover and Pressed (see design)

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @nicupop729 and @taherhagui, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Changing the link to your stylesheet to have ./src so it can be accessed easily during reviews.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (nicupop729) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (nicupop729) [reply index.html:292]: fixed

File: index.html [hunk 3]
```diff
@@ -268,6 +268,77 @@ <h4 class="experience-title">Skills</h4>
...
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea
+          maxlength="500"
+          name="message"
+          placeholder="Enter your text here"
+          required
+        ></textarea>
+        <button type="submit">Get in touch</button>
+      </form>
```
Comment (Mihndim2020) [inline index.html:292]: - [x] As requested by the former code reviewer, and as part of the project requirements, please make sure the "Get in touch" button styling: Active, Hover and Pressed (see design) is as specified in he design. 
- [OPTIONAL] Please the form has no styling, so is the button, it seems your styles are not properly linked, please kindly fix that. 
***The design***

wed2

***Your implementation***
yourswed1

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @nicupop729, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Require Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (nicupop729) [reply index.html:292]: done

**Review by carlos-ssh — APPROVED** [review]
Hi @nicupop729 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**